CHANGES

```diff
@@ -1,3 +1,8 @@
+2.0.0 (Aug 2017)
+* Fix some cases of non-compliance with the JsonRpc v2.0 specs (@kc284)
+* Check the structure of error objects of JsonRpc v2.0 is spec compliant (@kc284)
+* Add new function to retrieve the version of JSON-RPC alongside the Rpc.call from the request body (@kc284)
```

Comment:
This in reality is 'version and id' (the id bit was added at the top commit of this changelog)